hw2/02-search.js

```diff
@@ -1,3 +1,64 @@
-// Add your code here
 
-userInput.addEventListener('click', handleClick);
+function handleClick()
+{
+	console.log('Button clicked!');
+	const txt = document.getElementById('userInput');
+	console.log(txt.value);
+	
+	const items = document.querySelectorAll('.card');
+	for (let i = 0; i < items.length; i++) {
+        const card = items[i];
+		var title = card.querySelector('.card-title')
+		if(title.innerHTML.includes(txt.value))
```

Comment:
I think for usability, it would be nice to allow the user to enter in lowercase or uppercase and get the same result. You can achieve this by using toLowerCase() with your if condition.